content/tutorial/tutorial.md

```diff
@@ -1042,20 +1049,21 @@ Let's `map` over the `history` in the Game's `render` method:
   }
 ```
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/EmmGEa?editors=0010)**
+**[अभी तक का पूरा कोड देखें](https://codepen.io/gaearon/pen/EmmGEa?editors=0010)**
 
-For each move in the tic-tac-toes's game's history, we create a list item `<li>` which contains a button `<button>`. The button has a `onClick` handler which calls a method called `this.jumpTo()`. We haven't implemented the `jumpTo()` method yet. For now, we should see a list of the moves that have occurred in the game and a warning in the developer tools console that says:
+टिक-टैक्-टो के खेल के इतिहास में प्रत्येक चाल के लिए, हम एक सूची आइटम `<li>` बनाते हैं जिसमें एक बटन होता है `<button>`। बटन में एक  `onClick` हैंडलर होता है, जो कि `this.jumpTo()` नामक एक मेथड कॉल करता है। 
+हमने अभी तक `jumpTo()` मेथड बनाई नहीं की है। अभी के लिए, हमें खेल में होने वाली चालों की सूची दिखनी चाहिए और डेवलपर टूल्स कंसोल में एक चेतावनी दी गई है जो कहती है:
```

Comment:
check grammar of both sentences

Reply:
```
अभी के लिए, हमें खेल में हुई सारी चालें दिखनी चाहिए और एक ऐसी चेतावनी डेवलपर कंसोल में दिखनी चाहिए:
```

how about this?

Reply:
डेवलपर कंसोल में dikhegi

Reply:
Don't see this changed right now, might be a commit behind?

Reply:
> Don't see this changed right now, might be a commit behind?

Yes. I think is missed to commit this. Doing it right away